microsoft/ElectionGuard-SDK-C-Implementation
# Add .vscode tasks

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.



### Description

This PR adds .vscode tasks to make it easy to build and test in VS Code

#### To Test
1. pull the changes
2. run the default build task from the command palette
3. run the default test task from the command palette

Expected: builds and tests pass as expected

💔Thank you!
